### DISCUSSION TITLE
Select screen issue

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#220] [General]
when i use the select screen and check the checkbox beside the filter text box to select the whole list, it checks only the first checkbox in the list and leave all others unchecked although you have a text saying that all items selected ( on the right side of the filter text box).
Please advise.
Thanks.

**Comment by rejetto** [comment]
I just tried, and it works fine to me, marking all files.
Are you using any custom html?
You can provide a video, hoping I can spot some useful information, or maybe you can send me privately a link so that I can see the problem on your server: a@rejetto.com

**Reply by GeorgeGeorgeGe** [reply]
check your email please. Thanks.

**Comment by rejetto** [comment]
ok, i confirm that there's a bug limited to names with a space.
I'm working on it.

**Comment by rejetto** [comment]
would you try 0.44 beta? it should fix the problem
https://github.com/rejetto/hfs/discussions/221
